de.dlr.sc.virsat.server/src/de/dlr/sc/virsat/server/resources/ModelAccessResource.java

```diff
@@ -251,4 +294,28 @@ public Response getConcepts(@DefaultValue("true") @QueryParam(QP_ONLY_ACTIVE_CON
 		}
 	
 	}
+	
+	/**
+	 * Create a new Sei typed by the Se identified by the fullQualifiedName
+	 * @param fullQualifiedName of the sei type (se)
+	 * @param owner of the sei (either another sei or the repository for root seis)
+	 * @param editingDomain
+	 * @return uuid of the created sei
+	 */
+	public static String createSeiFromFqn(String fullQualifiedName, EObject owner, VirSatTransactionalEditingDomain editingDomain) {
+		ActiveConceptHelper helper = new ActiveConceptHelper(editingDomain.getResourceSet().getRepository());
+		
+		StructuralElementInstance newSei = StructuralFactory.eINSTANCE.createStructuralElementInstance();
+		
+		// Add the correct type using the fqn
+		int idx = fullQualifiedName.lastIndexOf(".");
+		Concept concept = helper.getConcept(fullQualifiedName.substring(0, idx));
+		StructuralElement se = ActiveConceptHelper.getStructuralElement(concept, fullQualifiedName.substring(idx + 1));

```

Comment:
I feel like the logic of geting the StructuralElement for an FQN should be part of the ActiveConceptHelper so we don't have those details in the server logic.